src/app/Scenes/MyCollection/Components/ConsignmentSubmissionStatus.tsx

```diff
@@ -0,0 +1,50 @@
+import { AlertCircleFillIcon, Flex, Text } from "@artsy/palette-mobile"
+import { ConsignmentSubmissionStatus_artwork$data } from "__generated__/ConsignmentSubmissionStatus_artwork.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface ConsignmentSubmissionStatusProps {
+  artwork: ConsignmentSubmissionStatus_artwork$data
+}
+
+const ConsignmentSubmissionStatus: React.FC<ConsignmentSubmissionStatusProps> = ({ artwork }) => {
```

Comment:
I'd vote for consistently using the term `submission` instead of `consignmentSubmission`or `consignment`
```suggestion
const SubmissionStatus: React.FC<ConsignmentSubmissionStatusProps> = ({ artwork }) => {
```